rskj-core/src/main/java/org/ethereum/core/TransactionReceipt.java

```diff
@@ -129,6 +145,8 @@ public Bloom getBloomFilter() {
         byte[] cumulativeGasRLP = RLP.encodeElement(this.cumulativeGas);
         byte[] gasUsedRLP = RLP.encodeElement(this.gasUsed);
         byte[] bloomRLP = RLP.encodeElement(this.bloomFilter.data);
+        byte[] result = RLP.encodeElement(this.executionResult);
+        byte[] err = RLP.encodeElement(this.error.getBytes(StandardCharsets.UTF_8));
```

Comment:
I can't find where EIP 658 specifies that there should be an error message,

Reply:
Note taken from another EIP:
[A recent EIP](https://github.com/ethereum/EIPs/pull/658) originally proposed adding return values to transaction receipts.  However, return data is not charged for (as it is not presently stored on the blockchain), so adding it to transaction receipts could result in DoS and spam opportunities. Instead, a simple Boolean `status` field was added to transaction receipts.  This was included in the Byzantium hard fork.

Reply:
I really really really can't find any EIP or formal specification of this change in the Internet.  
It's not even here:
https://github.com/ethereum/go-ethereum/blob/master/core/types/receipt.go

Reply:
I also could not find the spec that stated that we should use error. I just based my code on the PR of ethj. But we could find another way.

https://github.com/ethereum/ethereumj/blob/656301b4b146219b70b00ec4f9f8681583dd731c/ethereumj-core/src/main/java/org/ethereum/core/TransactionReceipt.java